src/NServiceBus.Msmq.AcceptanceTests/When_worker_sends_a_message_for_delayed_retry.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using AcceptanceTesting.Support;
+    using Config;
+    using Features;
+    using NServiceBus.Routing.Legacy;
+    using NUnit.Framework;
+    using ObjectBuilder;
+    using Transport;
+
+    public class When_worker_sends_a_message_for_delayed_retry : NServiceBusAcceptanceTest
```

Comment:
@timbussmann well the test validates if the message is received and I believe it is valid. The code in the core is wrong because when I debugged it, the message is actually deferred to the worker, not to the distributor. I've checked and it seems the problem is at least as old as 6.0.0-beta1.